SmartPlaces.Facilities/lib/IngestionManager/src/Exceptions/FacilitiesException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="FacilitiesException.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace IngestionManager.Exceptions
+{
+    using System;
+
+    public class FacilitiesException : Exception
```

Comment:
The root folder name for this sample is still SmartPlaces.Facilities, does that need to change too?

Reply:
This probably isn't the final repo where this code will reside. When we more it to its final resting location, we will update the path.